custom/icds_reports/views.py

```diff
@@ -768,12 +770,16 @@ def get(self, request, *args, **kwargs):
         config.update(get_location_filter(location, self.kwargs['domain']))
         loc_level = get_location_level(config.get('aggregation_level'))
 
-        data = []
+        data = {}
         if step == "map":
             if loc_level in [LocationTypes.SUPERVISOR, LocationTypes.AWC]:
                 data = get_prevalence_of_severe_sector_data(domain, config, loc_level, location, include_test)
             else:
-                data = get_prevalence_of_severe_data_map(domain, config, loc_level, include_test)
+                data = get_prevalence_of_severe_data_map(domain, config.copy(), loc_level, include_test)
+                sector = get_prevalence_of_severe_sector_data(
+                    domain, config.copy(), loc_level, location, include_test
```

Comment:
do you need to make two copies of `config` here? (and everywhere else in this file)

Reply:
@czue No, I will change this.